### PR TITLE
chore: update the kas-fleet-manager API to the latest version

### PR DESCRIPTION
**Pull request by wei-lee**
The kas-fleet-manager API is updated and some fields are renamed. In order to keep backward compatibility, the old fields are still returned by the API by marked as "deprecated". However, this causes a problem for the openapi generated as the generated field name is the same for both the old and new fields.

To solve this problem, a few openapi generate templates are added. The templates are based on the upstream ones (see https://github.com/OpenAPITools/openapi-generator/tree/v5.1.1/modules/openapi-generator/src/main/resources/go), with the additional changes to ignore deprecated fields. Another option here is to still keep the deprecated fields, but add a prefix (like `DeprecatedBootstrapServer`), but since the previous field name still exist, there is no need to keep the previous field. In the future when these deprecated fields are removed from the API, the templates can be removed too.

### CODE REVIEW

**Comment by wtrocki** [comment]
I like current solution as having templates our our side gives us much better flexibility.
Leaving review to @craicoverflow 

>  Another option here is to still keep the deprecated fields

Another option will be to include tags to deprecated API. Since SDK is still in development we can have:

- Default API 
- Depreciated API based on Depreciated tag

File: kafkamgmt/apiv1/client/docs/MetricsInstantQueryList.md [hunk 1]
```diff
@@ -1,108 +1,112 @@
...
+        ### NewMetricsInstantQueryList
 
-`func NewMetricsInstantQueryList() *MetricsInstantQueryList`
+        `func NewMetricsInstantQueryList() *MetricsInstantQueryList`
 
-NewMetricsInstantQueryList instantiates a new MetricsInstantQueryList object
-This constructor will assign default values to properties that have it defined,
-and makes sure properties required by API are set, but the set of arguments
-will change when the set of required properties is changed
+        NewMetricsInstantQueryList instantiates a new MetricsInstantQueryList object
```
Comment (craicoverflow) [inline kafkamgmt/apiv1/client/docs/MetricsInstantQueryList.md:18]: I guess the template is causing all the documentation to become indented?

Comment (wei-lee) [reply kafkamgmt/apiv1/client/docs/MetricsInstantQueryList.md:18]: yes, let me see if I can fix it.

Comment (wei-lee) [reply kafkamgmt/apiv1/client/docs/MetricsInstantQueryList.md]: This is resolved now.

**Comment by wei-lee** [comment]
@craicoverflow This is ready now.

**Review by craicoverflow — APPROVED** [review]
I'm seeing some places where there are extra blank lines in the generated markdown files, but this is not affecting rendering in any way, so I;m happy to approve 👍🏻 

Thanks @wei-lee ! I will merge once you get back to me to indicate whether you wanted to resolve those line breaks.

**Comment by wei-lee** [comment]
@craicoverflow I did look into the new lines issue but couldn't figure out where the extra line is introduced. I compared our templates against the default ones and the line numbers matches. So I didn't spend too much time looking into this as you said, not a big issue. So if you can merge this that will be great.
Thanks.

**Pull request merged**